### PR TITLE
Makes RND Autolathe Printable

### LINKED ISSUE #9: A way to get an omnilathe/research from an Autolathe.

### PR DESCRIPTION
**Pull request by unit0016**
## About The Pull Request
Closes #9 

You can now print out a full RND kit anywhere with an autolathe. They cost one sheet of cardboard and four sheets of glass on T4 parts, scaling up to a little under two times that on T1.

As an aside, the TGUI interface for autolathes never expected cardboard to be required for any recipes, so displays as a `(?)`. It does function as intended despite this and's an upstream issue.

# Will change cost to something less batshit insane if asked, just figured ships didn't survive long enough to make much pickaxe mining practical.
## Why It's Good For The Game
The request in question
## Changelog
:cl:
add: RND Kits are now autolathe printable.
/:cl:

### CODE REVIEW

**Comment by unit0016** [comment]
actually this probably shouldn't be in the edits folder. moving it to base

**Comment by JohnFulpWillard** [comment]
My concern is the cardboard, I dunno how easy it is for ships to get, autolathe not displaying it in its UI should be easy to fix

File: voidcrew/modules/research/autolathe_designs.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/datum/design/research
+	name = "Research & Development Kit"
+	id = "rndkit"
+	build_type = AUTOLATHE
+	materials = list(/datum/material/cardboard = 2000, /datum/material/glass = 4000) // The materials for one box + all boards inside exactly.
+	build_path = /obj/item/storage/box/rndboards
+	category = list(
+		RND_CATEGORY_INITIAL,
+		RND_CATEGORY_CONSTRUCTION + RND_SUBCATEGORY_CONSTRUCTION_MACHINERY
```
Comment (ZephyrTFA) [inline voidcrew/modules/research/autolathe_designs.dm:9]:
```suggestion
		RND_CATEGORY_CONSTRUCTION + RND_SUBCATEGORY_CONSTRUCTION_MACHINERY,
```

**Comment by ZephyrTFA** [comment]
five years later

**Pull request merged**